NBitcoin.Altcoins/Mogwai.cs

```diff
@@ -0,0 +1,237 @@
+﻿using NBitcoin;
+using NBitcoin.Crypto;
+using NBitcoin.DataEncoders;
+using NBitcoin.Protocol;
+using NBitcoin.RPC;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NBitcoin.Altcoins
+{
+	// Reference: https://github.com/mogwaicoin/mogwai/blob/master/src/chainparams.cpp
+	public class Mogwai : NetworkSetBase
+	{
+		public static Mogwai Instance { get; } = new Mogwai();
+
+		public override string CryptoCode => "MOG";
+
+		private Mogwai()
+		{
+
+		}
+
+		public class MogwaiConsensusFactory : ConsensusFactory
+		{
+			private MogwaiConsensusFactory()
+			{
+			}
+
+			public static MogwaiConsensusFactory Instance { get; } = new MogwaiConsensusFactory();
+
+			public override BlockHeader CreateBlockHeader()
+			{
+				return new MogwaiBlockHeader();
+			}
+			public override Block CreateBlock()
+			{
+				return new MogwaiBlock(new MogwaiBlockHeader());
+			}
+		}
+
+#pragma warning disable CS0618 // Type or member is obsolete
+		public class MogwaiBlockHeader : BlockHeader
```

Comment:
you do not need this. Mogwai block format is same as bitcoin

Reply:
okay, thank you for the hint I removed it in latest commit. It was copy paste from dash ... :-)